### PR TITLE
Adding project quota to the datatable

### PR DESCRIPTION
**Pull request by JaymeeH**
adding storage capacity and usage to the dashboard presenter
passing the session id through the welcome controller
refs #1030

### CODE REVIEW

**Comment by JaymeeH** [comment]
Screenshot 2024-11-27 at 1 47 28 PM

**Comment by JaymeeH** [comment]
Screenshot 2024-12-03 at 4 45 31 PM

**Comment by JaymeeH** [comment]
Screenshot 2024-12-04 at 1 47 16 PM

File: app/models/project.rb [hunk 1]
```diff
@@ -240,15 +240,14 @@ def storage_capacity_xml
   end
 
   def storage_capacity(session_id:)
-    requested_capacity = storage_capacity_xml
```
Comment (carolyncole) [inline app/models/project.rb:243]: Can you also clean up the storage_capacity_xml function? https://github.com/pulibrary/tigerdata-app/blob/26bf465c12a856282cbd10065a0ade1b3a11d70d/app/models/project.rb#L237

File: app/presenters/project_dashboard_presenter.rb [hunk 2]
```diff
@@ -40,4 +40,13 @@ def role(user)
...
     end
   end
+
+  def quota_usage(session_id:)
+    quota_usage = "#{project.storage_usage(session_id:)} out of #{project.storage_capacity(session_id:)} used"
+    quota_usage
+  end
+
+  def quota_percentage(session_id:)
+    # figure out how to calculate percentage of storage used with different and unpredictable units
```
Comment (carolyncole) [inline app/presenters/project_dashboard_presenter.rb:50]: I think storage is now a hash.  I wonder if there needs to be some updates to this code.  https://github.com/pulibrary/tigerdata-app/blob/main/app/models/project_metadata.rb#L149

**Review by carolyncole — CHANGES_REQUESTED** [review]
@JaymeeH nice work.  It looks like the percentage is not getting calculated.  Was this intended? I also wonder if the fact that the size is a hash instead of a string will cause some issues. 

At the very least can you clean up the method you made unused?

**Review by carolyncole — APPROVED** [review]
Thanks @JaymeeH!  Nice step towards the finished product.

**Pull request merged**